### ISSUE TITLE
When camera or player does not move, enemy's position is not reflected in MiniMap

### ISSUE THREAD

**Issue by nyuntaka** [RafaelBarbosatec/bonfire#517]
### What happened?

https://github.com/RafaelBarbosatec/bonfire/assets/14065274/3c2f72b0-b3f3-4190-a180-00472102c7b8

When camera or player does not move, enemy's position is not reflected in MiniMap.
It will be reflected the moment the player moves.

Note: componentsRender is also not called

### Steps to reproduce?

```Markdown
1. Open the "Bonfire Example".
https://bonfire-engine.github.io/#/doc/examples?id=bonfire-example

2. Select the "Map by tiled" in "Mini games".

3. Move the player closer to an enemy.

4. Look at the Mini map.

*In addition to the sample, I also checked my own code.
```

### What did you expect to happen?

When an enemy moves, reflect its position on the Mini map.

### Bonfire version

3.9.0

**Comment by RafaelBarbosatec** [comment]
fixed in version 3.9.2

**Comment by nyuntaka** [comment]
I was able to confirm that it was fixed.
Thank you very much.
